src/api/fs.go

```diff
@@ -1,17 +1,22 @@
 package api
 
 import (
+	"github.com/nasz-elektryk/spito/shared"
 	"os"
 	"regexp"
 	"strings"
 )
 
-func PathExists(path string) bool {
+type FsApi struct {
+	ImportLoopData *shared.ImportLoopData
+}
+
+func (*FsApi) PathExists(path string) bool {
```

Comment:
Please don't simulate namespaces.

Reply:
But we need to somehow provide `FsVRCT` instance in order to make any changes.
I used `FsApi` in order to provide the mentioned instance.